module-db/Interface/ContactRecord.cpp

```diff
@@ -1595,3 +1606,24 @@ auto ContactRecordInterface::changeNumberRecordInPlaceIfCountryCodeIsOnlyDiffere
     }
     return true;
 }
+auto ContactRecordInterface::hasContactRecordSameNumbers(const ContactRecord &rec) -> bool
+{
+    if (rec.numbers.size() >= 2) {
+        if (rec.numbers.size() > 2) {
+            LOG_WARN("Contact record has more tha 2 numbers. Checking similarity for first 2 numbers only");
```

Comment:
Spelling error:
"The contact record has more than 2 numbers. Checking similarity for the first 2 numbers only"

BTW. Is it possible to have more than two numbers? Maybe we should handle it somehow and return an error because the user cannot add more than two numbers while creating a new contact.

Reply:
I would consider doing a precheck in Add and Update functions 
```
if(rec.numbers.size() > 2)
{
LOG_ERROR("Contact can not have more than two numbers");
return false;
}
```

Reply:
Practically the user are not able to provide more than 2 numbers for Contact. Technically is there a possibility to add more, but only the GUI stands in the way.

I'll add this check, as you wrote, but without return and as warning kind of log, because in code are many "leftovers" which wanted to provide more advanced functions to Contacts, like type of Number (ContactNumberType::HOME/ WORK etc.)
And because are code is Open I wouldn't stay in the way for other implementation of MuditaOS  than ours (for Pure).